main.rb

```diff
@@ -0,0 +1,13 @@
+require_relative 'mylist'
+require_relative 'enumerable'
```

Comment:
- [ ] [optional] Great work adding the tests for your class, however, I would recommend you import the enumerable module in MyList class instead of here, then only import the class to test its functionality.